### PR TITLE
Add title and description to tags and tag pages

### PR DESCRIPTION
**Pull request by askvortsov1**

### CODE REVIEW

**Comment by askvortsov1** [comment]
This fully covers #119, which can probably be closed if this is accepted.

**Comment by luceos** [comment]
@askvortsov1 linking issues/pr's can only be done from the body (not the subject) unless it's in the same repo (I think), so: https://github.com/flarum/tags/pull/72

**Comment by askvortsov1** [comment]
> @askvortsov1 linking issues/pr's can only be done from the body (not the subject) unless it's in the same repo (I think), so: [flarum/tags#72](https://github.com/flarum/tags/pull/72)

Thanks!

File: locale/flarum-tags.yml [hunk 1]
```diff
@@ -89,6 +89,12 @@ flarum-tags:
       removed_tags_text: "{username} removed the {tagsRemoved}."
       tags_text: "{tags} tag|{tags} tags"
 
+    # These tags are used in meta elements of the forum.
+    meta:
+      tag_description: "All discussions with the {tag} tag"
+      tags_title: => flarum-tags.ref.tags
+      tags_description: "All Tags"
```
Comment (franzliedke) [inline locale/flarum-tags.yml]: Please order the keys alphabetically.

Comment (askvortsov1) [reply locale/flarum-tags.yml]: Fixed!

**Comment by askvortsov1** [comment]
> Thank you!
> 
> Please include at least the context of what you're changing in the PR title. Cross references can go in the description.

Noted, will do going forward!

**Comment by franzliedke** [comment]
I've tweaked the keys a bit, but used your texts, all in master.

When you have the text, feel free to read our extensive documentation on internationalization, especially [this handy appendix](https://flarum.org/docs/extend/i18n.html#appendix-a-standard-key-format). :grinning: 

It's the awesome legacy of our former internationalization expert, who unfortunately can no longer contribute.

**Pull request closed without merging**